front/pages/w/[wId]/a/[aId]/runs/[runId]/index.tsx

```diff
@@ -43,7 +43,7 @@ export const getServerSideProps = withDefaultUserAuthRequirements<{
 
   const isBuilder = auth.isBuilder();
 
-  const app = await getApp(auth, context.params?.aId as string);
+  const app = await AppResource.fetchById(auth, context.params?.aId as string);
```

Comment:
missing toJSON line 53

Reply:
TS duck typing makes it work `AppResource` shares all attributes of `AppType`... Also the `toJSON` helps here as it's implicitly call when seriaized.

Reply:
(fixed)